pkg/controller/controller_test.go

```diff
@@ -558,6 +558,17 @@ func TestGetSecretReference(t *testing.T) {
 			pvc:          nil,
 			expectRef:    &v1.SecretReference{Name: "name", Namespace: "ns"},
 		},
+		"simple - valid, pvc name and namespace": {
+			secretParams: provisionerSecretParams,
+			params:       map[string]string{provisionerSecretNameKey: "name", provisionerSecretNamespaceKey: "ns"},
```

Comment:
Can you make the keys used in the params different from the values in the PVC? That way we make sure we're not accidentally using pvc values.